TTS/tts/datasets/TTSDataset.py

```diff
@@ -39,6 +39,7 @@ def __init__(self,
             compute_linear_spec (bool): compute linear spectrogram if True.
             ap (TTS.tts.utils.AudioProcessor): audio processor object.
             meta_data (list): list of dataset instances.
+            tp (dict): dict of custom text characters used for converting texts to sequences.
```

Comment:
Also a different PR, but since you already try to document types here, you could use python3.6+ [typing hints](https://docs.python.org/3/library/typing.html), which can be type checked with mypy.

Reply:
that requires passing over the whole project and I am really lazy looking to do that. But eventually, we need it I guess.